src/Node/Expression/SpreadExpression.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Node\Expression;
+
+use Twig\Compiler;
+
+final class SpreadExpression extends AbstractExpression
+{
+    public function __construct(AbstractExpression $node, int $lineno)
+    {
+        parent::__construct(['node' => $node], [], $lineno);
+    }
+
+    public function compile(Compiler $compiler): void
+    {
+        $compiler
+            ->raw('...')
```

Comment:
shouldn't we enforce putting a space before the `...` to avoid issues in compiled code about where it is associated (as done in AbstractUnary) ?
Btw, maybe this could extend `AbstractUnary`

Reply:
Done